phaseshifts/phsh.py

```
@@ -857,15 +878,28 @@ def _fatal(err):
+
     # create phase shifts (warning: black magic within - needs testing)
     if verbose:
-        print("Phase shift auto-generation parameters")
-        print("\tbulk input file: {}".format(args.bulk))
-        print("\tslab input file: {}".format(args.slab))
-        print("\tformat: {}".format(args.format))
-        print("\tbackend: {}".format(args.backend))
-        print("\tlmax: {}".format(args.lmax))
-        print("\trange: {} eV".format([s for s in args.range]))
+        sys.stdout.write("Phase shift auto-generation parameters\n")
+        sys.stdout.write("\tbulk input file: {}\n".format(args.bulk))
```

Ruff (0.14.10)

Use f-string instead of `format` call

Convert to f-string

(UP032)